providers/go-feature-flag/pom.xml

```diff
@@ -62,6 +62,12 @@
             <version>33.3.1-jre</version>
         </dependency>
```

Comment:
**suggestion:** Since `guava` is not used anymore we should probably remove the dependency.

Reply:
Actually the reason I did not remove the dependency is because `HttpHeaders` from the package `com.google.common.net` is being used on `GoFeatureFlagController`, which comes from Guava.

Reply:
> Actually the reason I did not remove the dependency is because `HttpHeaders` from the package `com.google.common.net` is being used on `GoFeatureFlagController`, which comes from Guava.

In that case we should probably set as dependency `com.google.common.net` instead of guava or just replace the constants by our own constants.

Reply:
Ok, I'm working on that